src/main/java/com/easypost/model/PaymentMethodObject.java

```diff
@@ -172,7 +172,7 @@ public PaymentMethodType getType() {
      *
      * @return the endpoint for this PaymentMethodObject.
      */
-    String getEndpoint() {
+    public String getEndpoint() {
```

Comment:
I hate that Java doesn't have a scope where you can use a function in your own package but block it from outsiders just because it's in a different folder...